### PR TITLE
[Feature] Ceph health CheckCommand

### PR DESCRIPTION
**Pull request by miso231**
Add ceph plugin to itl

### CODE REVIEW

File: doc/10-icinga-template-library.md [hunk 1]
```diff
@@ -3062,6 +3062,24 @@ glusterfs_disk_critical    | **Optional.** Return a critical error if disk usage
...
 
+#### ceph <a id="plugins-contrib-command-ceph"></a>
+
+The [ceph plugin](https://github.com/ceph/ceph-nagios-plugins)
+is used to check the Ceph storage health on the server.
+
+Custom attributes passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                       | Description
+---------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```
Comment (Crunsher) [inline doc/10-icinga-template-library.md]: Please remove the excess dashes at the end, there should be as many to go over `**Optional.**`. Other tables do this incorrectly too, but it's not much of an issue.

**Comment by Crunsher** [comment]
Going to merge this once we have 2.9.2 out of the way.

File: itl/plugins-contrib.d/storage.conf [hunk 2]
```diff
@@ -56,3 +56,49 @@ object CheckCommand "glusterfs" {
 	vars.glusterfs_inode_warning = 90
 	vars.glusterfs_inode_critical = 95
 }
+
+object CheckCommand "ceph" {
+	command = [ PluginDir + "/check_ceph.py" ]
+
+	arguments = {
+		"-e" = {
+			value = "$service.vars.ceph_exec_dir$"
```
Comment (mcktr) [inline itl/plugins-contrib.d/storage.conf]: Following other CheckCommands in the ITL, the _value_ attribute should be only `ceph_exec_dirs`. This way one can use `vars.ceph_exec_dirs` as a custom variable  to adjust the attribute. 

Same for the other argument value attributes.

Comment (Crunsher) [reply itl/plugins-contrib.d/storage.conf]: Good catch, must have been a caffeine underdose on my part ^_^

**Comment by dnsmichi** [comment]
Thanks 👍

**Pull request merged**